config/jail.conf

```diff
@@ -839,3 +839,14 @@ logencoding = utf-8
 # See "haproxy-http-auth" filter for a brief cautionary note when setting
 # maxretry and findtime.
 logpath  = /var/log/haproxy.log
+
+
+[zoneminder]
+# Zoneminder HTTP/HTTPS web interface auth
+# Logs auth failures to apache2 error log
+enabled = false
+port    = http,https
+filter  = zoneminder
+logpath = /var/log/apache*/*error.log
```

Comment:
use `%(apache_error_log)s` instead so could be overloaded in a single place per distribution